src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  border-radius: 4px;
+}
+
+.search-bar,
+.search-bar__field,
+.search-bar__input {
+  width: 100%;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.search-bar__big {
+  margin: 20px 0;
+  height: 70px;
+}
```

Comment:
Write all styles for big input to `search-bar` class and then override sizes for small input

Reply:
Fix for all elements